### ISSUE TITLE
any limit to join a new node or how to build a private nodes group?

### ISSUE THREAD

**Issue by aohan237** [n0-computer/iroh-blobs#168]
i'm new , read all the docs,but no information about this.

it seems that any node will automatic join, but how you decide which node should join or how can you limit the node to join

such as add an group key , or something like this

or say, every one we who use iroh to setup a node can see each other?

thanks in advance

**Comment by dignifiedquire** [comment]
I don't think this is really actionable, as this includes questions around iroh-cli which does not exist anymore. Please feel free to open a new issue with more details and what you are trying to achieve given the current software state.
